provisioning/miniprov/hare_mp/main.py

```diff
@@ -310,39 +310,51 @@ def kv_cleanup():
         raise RuntimeError('Error during key delete in transaction')
 
 
+def pre_factory():
+    logging.info('Executing pre-factory cleanup command...')
+    deployment_logs_cleanup()
```

Comment:
We need to cleanup /var/motr/ as well.
Also I think Let's delete `/etc/sysconfig/m0d-0x7200000000000001*` and `/etc/sysconfig/s3server-0x7200000000000001*`

Reply:
@mssawant https://seagate-systems.atlassian.net/wiki/spaces/~527368708/pages/edit-v2/563872735?draftShareId=d5838841-79a4-4353-9d31-2e149c19c640
According to above doc, these changes will be taken care by Motr.
Is my understanding correct?

Reply:
@SwapnilGaonkar7, As per the document, motr is cleaning `/var/motr/m0d-*`, so we need to cleanup up `/var/motr/hax` and sysconfig file as mentioned above.

Reply:
We are cleaning  `/var/motr/m0d-*` and `/var/motr/hax`  in  [EOS-23169](https://jts.seagate.com/browse/EOS-23169)  this story. So, I think we can do it in the next task.  Is this okay?
cc. @mssawant @SwapnilGaonkar7 @vaibhavparatwar

Reply:
May be we can do this in a separate ticket. cc @vaibhavparatwar